projects/js-packages/charts/agents.md

```diff
@@ -0,0 +1,158 @@
+# AGENTS.md
+
+This file provides AI coding agents with specific instructions and context for working on the @automattic/charts library.
+
+## Project Overview
+
+**@automattic/charts** is a React/TypeScript charting library built for interactive data visualizations within Automattic products. The library is built on top of `@visx/xychart` and emphasizes accessibility, responsiveness, and developer experience through a component composition API.
```

Comment:
I think it's only partially  built on `@visx/xychart`  - guess we could say it's built on Airbnb visx?